src/main/java/com/cloudbees/jenkins/plugins/docker_build_env/Docker.java

```diff
@@ -29,13 +29,15 @@
     private final String dockerExecutable;
     private final DockerServerEndpoint dockerHost;
     private final boolean verbose;
+    private final boolean privileged;
 
-    public Docker(DockerServerEndpoint dockerHost, String dockerInstallation, AbstractBuild build, Launcher launcher, TaskListener listener, boolean verbose) throws IOException, InterruptedException {
+    public Docker(DockerServerEndpoint dockerHost, String dockerInstallation, AbstractBuild build, Launcher launcher, TaskListener listener, boolean verbose, boolean privileged) throws IOException, InterruptedException {
```

Comment:
Causes a binary compatibility issue. Please keep the constructor with the original signature